conda_smithy/lint_recipe.py

```diff
@@ -149,6 +151,15 @@ def lintify(meta, recipe_dir=None):
     except RuntimeError as e:
         lints.append(str(e))
 
+    # 13: Subheaders should be in the allowed subheadings
+    for section in major_sections:
+        expected_subsections = FIELDS[section]
```

Comment:
Also what's up with that syntax highlighting on GitHub? Is `FIELDS` some special Python name that I have never heard of? 😕